extension/export_HartreeFockProvider.cc

```diff
@@ -408,79 +410,110 @@ void export_HartreeFockProvider(py::module& m) {
   py::class_<HartreeFockProvider, std::shared_ptr<HartreeFockProvider>,
              PyHartreeFockProvider>(
         m, "HartreeFockProvider", hfdata_i,
-        "Abstract class defining the interface for passing data from the host program to "
-        "adcc. All functions of this class need to be overwritten explicitly from "
-        "python.\nIn the remaining documentation we denote with `nf` the value returned "
+        "Abstract class defining the interface for passing data from the host "
+        "program to "
+        "adcc. All functions of this class need to be overwritten explicitly "
+        "from "
+        "python.\nIn the remaining documentation we denote with `nf` the value "
+        "returned "
         "by `get_n_orbs_alpha()` and with `nb` the value returned by "
         "`get_nbas()`.")
         .def(py::init<>())
         //
         .def("get_conv_tol", &HartreeFockProvider::get_conv_tol,
-             "Returns the tolerance value used for SCF convergence. Should be roughly "
+             "Returns the tolerance value used for SCF convergence. Should be "
+             "roughly "
              "equivalent to the l2 norm of the Pulay error.")
         .def("get_restricted", &HartreeFockProvider::get_restricted,
              "Return *True* for a restricted SCF calculation, *False* otherwise.")
         .def("get_energy_scf", &HartreeFockProvider::get_energy_scf,
-             "Returns the final total SCF energy (sum of electronic and nuclear terms.")
+             "Returns the final total SCF energy (sum of electronic and nuclear "
+             "terms.")
         .def("get_spin_multiplicity", &HartreeFockProvider::get_spin_multiplicity,
-             "Returns the spin multiplicity of the HF ground state. A value of 0* (for "
+             "Returns the spin multiplicity of the HF ground state. A value of "
+             "0* (for "
              "unknown) should be supplied for unrestricted calculations.")
         .def("get_n_orbs_alpha", &HartreeFockProvider::get_n_orbs_alpha,
-             "Returns the number of HF *spin* orbitals of alpha spin. It is assumed the "
-             "same number of beta spin orbitals are used. This value is abbreviated by "
+             "Returns the number of HF *spin* orbitals of alpha spin. It is "
```

Comment:
The line breaks here (and below) need to be cleaned up.

Reply:
Yes... Do you have the appropriate `.clang-format` file for this?

Reply:
Ah ... true, it's not committed ... let me work on that.

Reply:
No it is committed :smile: ... see the `extension` subfolder ... and I checked for me it does not make those breaks (using clang-format 8)

Reply:
My `clang-format` (also version 8) seems to ignore the `.clang-format` file 😦 
I assume running
```
clang-format -style=file -i bla.cc
```
is correct?

Reply:
Usually it loads the file from the folder automatically, so `-style=file` should not even be needed, but otherwise it's correct as far as I know.

Reply:
I'm not able to fix this, unfortunately... probably _you_ can run `clang-format` before we merge this PR.

Reply:
I don't mind doing the formatting here, but can you undo your formatting changes to these files first? Then I just have to run `clang-format -i` and commit.

Reply:
Reverted.